R/extract.R

```diff
@@ -565,11 +638,26 @@ extract.broom <- function(model, ...) {
   return(tr)
 }
 
+#' \code{\link{extract}} method for \code{broom} objects
+#'
+#' \code{\link{extract}} method for \code{broom} objects. These objects are
+#' created by the \code{\link[ANY]{broom}} function in the \pkg{ANY} package.
```

Comment:
The package isn't called `ANY`. It's the `broom` package, I think. Can you fix that and add that this method is used by default if no other suitable `extract` method is found?

Reply:
I tried to address this already, but for some reasons I'm not undertanding, roxygen is still printing ANY. I think it dependes on the fact that is set method it only says ANY. It is different from every other one. 

```
setMethod("extract", signature = className("ANY"),
          definition = extract.broom)
```

I'll see if I can fix it.

Reply:
nope, it's the signature that generates the file name. As long as the class is `ANY` I cannot do anything to change it in Roxygen. 

Using 
```
setMethod("extract", signature = className("broom","ANY"),
          definition = extract.broom)
```
it corrects the error. However, it might not work as extract function. I'm trying to test it but I can't think of a model that requires broom right now.

Reply:
`ANY` just means it should be used no matter what the class name as, as long as there is no other, more specific extract method available. There is no need to change the `setMethod` command as this is the desired behavior. It's also fine that Roxygen creates this file etc. I just want you to change the one sentence in your description because there is no `ANY package`...